docs/Rootless.md

```diff
@@ -33,19 +35,32 @@ endif
 ```
 
 - The iOS 14 arm64e ABI mentioned in [arm64e Deployment](arm64e-Deployment.html) is now *required* for the relevant devices
-    - Currently, it's only possible to build for the new ABI on macOS
-        - If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
+    - Currently, it's only possible to build for the new ABI on macOS (as only the Xcode tools can build for it)
+        - Unfortunately, new toolchain does not support building for the old ABI. If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
```

Comment:
```suggestion
        - Unfortunately, this new ABI lacks backwards compatibility with the old ABI. If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
```

Reply:
Here I think is wrong, because the point isn't that the ABIs aren't inter-compatible (as that's the whole primary point), but that the new compiler cannot build for the old ABI [apparently], which is something one might expect such as being able to cross-compile for different architectures with some switches...

Reply:
While the issue may spawn from the abilities (or lack thereof) of the newer ld64, my thought process was that Apple doesn't seem like they would allow two arm64e slices, so the ABIs would be mutually exclusive. In that case, the issue arises due to their lack of compatiblity with each other.

Reply:
Yeah but the solution proposed here is to build just for the older ABI for rootful and newer for rootless, rather than have two slices at once (but the newer toolchain is now incapable of producing older ABI for such thing)

Reply:
I see your point. In that case...
```suggestion
        - Unfortunately, this newer toolchain does not support building for the old ABI. If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
```